docs/shared_parsers_catalog/neutron_server_log.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Server Log
+==================
+
+NeutronServerLog - ``/var/log/neutron/server.log``
+--------------------------------------------------
+.. autoclass:: insights.parsers.neutron_server_log.NeutronServerLog
+   :members:
+   :show-inheritance:
+
```

Comment:
@PaulWay I don't believe there is really much benefit in adding this complexity when there is only a single mapper in the module.